airflow-core/docs/core-concepts/auth-manager/index.rst

```diff
@@ -172,6 +172,40 @@ cookie named ``_token`` before redirecting to the Airflow UI. The Airflow UI wil
 .. note::
     Do not set the cookie parameter ``httponly`` to ``True``. Airflow UI needs to access the JWT token from the cookie.
 
+Refreshing JWT Token
+''''''''''''''''''''
+The refresh token ``<METHOD> <PATH>`` is ``POST /auth/token/refresh``. It returns a new JWT token in the cookie where updated token is stored.
```

Comment:
I do not think it is needed to mention the path, each auth manager can come up with their own path. It does not really matter because Airflow can get this path with `get_url_refresh`. By the way, it would be great to mention this method in the doc, saying that if the auth manager provide an endpoint to refresh the token, it should be returned with `get_url_refresh`